src/swagger-specs/flow-service.yaml

```diff
@@ -1032,88 +975,88 @@ definitions:
     properties:
       specName:
         type: string
-        example: "Basic Authentication for a cloud storage source"
+        example: "Basic Authentication for google-cloud"
```

Comment:
Should Google be capitalized?

Reply:
in theory yes, but in this case, "Basic Authentication for google-cloud" is the actual coded name of the specName so not this one